docs/utility_methods/exception_handlers.md

```diff
@@ -0,0 +1,40 @@
+# Exception handlers
+
+_Added in v0.5.7_
+
+Exceptions handlers are function that can be assigned to your app to handle exceptions that occur during the application runtime.
+They are useful for customizing the response when an error occurs, logging errors, and performing cleanup tasks.
+
+## Types
+
+Reflex support two type of exception handler `frontend_exception_handler` and `backend_exception_handler`.
+
+They are both used to handle exceptions that occur in the `frontend` and `backend` respectively.
```

Comment:
remove "both". Maybe explain a bit what a frontend error is compared to a backend. A frontend is a Javascript error, while a backend is an error in the event handler